aws-amplify/amplify-android
# feat: Add EventSubmitter to Pinpoint

Issue #, if available:

Description of changes:
feat: Add EventSubmitter to Pinpoint

How did you test these changes?

Manually tested

[ *] Added Unit Tests

Documentation update required?

[* ] No
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
